README.md

```diff
@@ -8,6 +8,7 @@ SkyWalking Infra E2E is the next generation End-to-End Testing framework that ai
 
 ## Documentation
 
+- **[Official documentation](docs/README.md)**
```

Comment:
Official doc should be controlled by website

Reply:
Read https://github.com/apache/skywalking-website/blob/master/data/docs.yml

Reply:
We could have latest and per version documentations.

Reply:
Sure, I could create another PR to add content to the website project, but for now, only have the latest documentation.

Reply:
Latest document is able to go to website. Check this, https://github.com/apache/skywalking-website/blob/master/data/docs.yml#L11. It is going to be updated every day base on the auto build.

Reply:
I have deleted the document selection, after this PR merged, I will add the official document link to the website, and after that, I could add the website link into Infra E2E.